lib/game.rb

```diff
@@ -48,35 +48,12 @@ def player_move
     @end_round = false
   end
 
-  def win_hor
-    if @board_game.board[0] == @board_game.board[1] && @board_game.board[0] == @board_game.board[2] ||
-       @board_game.board[3] == @board_game.board[4] && @board_game.board[3] == @board_game.board[5] ||
-       @board_game.board[6] == @board_game.board[7] && @board_game.board[6] == @board_game.board[8]
-      true
-    end
-  end
-
-  def win_dig
-    if @board_game.board[0] == @board_game.board[4] && @board_game.board[0] == @board_game.board[8] ||
-       @board_game.board[2] == @board_game.board[4] && @board_game.board[2] == @board_game.board[6]
-      true
-    end
-  end
-
-  def win_ver
-    if @board_game.board[0] == @board_game.board[3] && @board_game.board[0] == @board_game.board[6] ||
-       @board_game.board[1] == @board_game.board[4] && @board_game.board[1] == @board_game.board[7] ||
-       @board_game.board[2] == @board_game.board[5] && @board_game.board[2] == @board_game.board[9]
-      true
-    end
-  end
-
   def draw?
     false if @board_game.board.all? { |idx| idx.is_a?(Integer) }
   end
```

Comment:
Not working.
It didn't detect the draw at that state of the game
O  X  O
O  X  X
X  O  O

Reply:
Also, it could also move into the `Board` class

Reply:
Hey, thank you for your feedback.

I was having a little trouble in making those changes so I schedule today a meeting with a TSE. For what we could understand I should have put gets.chomp and puts outside the logic of player move so I moved the gets chomp to player file and define a method to have only the message for the player.